rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -631,6 +645,9 @@ def load(
                     vocabulary=vocabulary,
                 )
 
+            for v in vectorizers.values():
+                v.vocabulary_ = v.vocabulary
```

Comment:
Otherwise loading did not work for me. `CountVectorizer` has two vocabulary variables: `vocabulary` and `vocabulary_`. When we load it only `vocabulary` is filled. However, we expect `vocabulary_` to be filled as we need it later.

Reply:
huh, why does it work on master?

Reply:
not sure... can verify it again

Reply:
it is working on master, we added the check `if self._check_attribute_vocabulary(attribute):` in the process method. This failed if `vocabulary_` is not set. But the check is not really needed. Trying to remove it right now.

Reply:
updated

Reply:
they do it in transform: https://github.com/scikit-learn/scikit-learn/blob/1495f69242646d239d89a5713982946b8ffcf9d9/sklearn/feature_extraction/text.py#L1106

not sure what is the best approach for us

Reply:
it is working, we don't need it when processing, we just need the `vocabulary_` set during training time.